schema/schema.py

```diff
@@ -22,20 +30,22 @@ def format_param(param: click.core.Option) -> Dict[str, Union[Optional[str], Lis
                        a list of those allowed options
     * default - The default option for that parameter
     """
-    formatted_param: Dict[str, Union[Optional[str], List[str]]] = {"name": param.name, "help": param.help}
+    formatted_param: Dict[str, Any] = {"title": param.name, "description": param.help}
 
     # NOTE: Params do not have explicit "string" type; either "text" or "path".
     #       All choice options are from a set of strings.
-    if param.type.name in ["text", "path", "choice"]:
+    if param.type.name.lower() in ["text", "path", "choice", "filename", "directory"]:
         formatted_param["type"] = "string"
+    elif param.type.name.lower() == "list":
+        formatted_param["type"] = "array"
     else:
-        formatted_param["type"] = param.type.name
+        formatted_param["type"] = param.type.name.lower() or "string"
```

Comment:
nit: can this be a variable instead of using `.lower()` three times? That way if something were to change, we can just change one line.